File: CSharp/fizzBuzz.cs
using System;

class Program {
    static void FizzBuzz(int n, int x, int y) {
        for (int i = 1; i <= n; i++) {
            if (i % x == 0 && i % y == 0) {
                Console.WriteLine("FizzBuzz");  // Ambos
            } else if (i % x == 0) {
                Console.WriteLine("Fizz");      // Múltiplo de x
            } else if (i % y == 0) {
                Console.WriteLine("Buzz");      // Múltiplo de y
            } else {
                Console.WriteLine(i);           // Não é múltiplo
            }
        }
    }

    static void Main() {
        FizzBuzz(10, 3, 5);  // Exemplo de chamada padrão
    }
}

File: C/fizzBuzz.c
#include <stdio.h>

void fizzBuzz(int n, int x, int y) {
    for (int i = 1; i <= n; i++) {
        if (i % x == 0 && i % y == 0) {
            printf("FizzBuzz\n");  // Ambos
        } else if (i % x == 0) {
            printf("Fizz\n");      // Múltiplo de x
        } else if (i % y == 0) {
            printf("Buzz\n");      // Múltiplo de y
        } else {
            printf("%d\n", i);     // Não é múltiplo
        }
    }
}

int main() {
    fizzBuzz(10, 3, 5);  // Exemplo de chamada padrão
    return 0;
}